### PR TITLE
DOCS-3507 Mention nconnect support on SCALE NFS

### PR DESCRIPTION
**Pull request by tonyriv3**

### CODE REVIEW

File: content/SCALE/Shares/_index.md [hunk 1]
```diff
@@ -901,6 +901,11 @@ After installing the module, connect to an NFS share by entering `sudo mount -t
 In the above example, *{IPaddressOfTrueNASsystem}* is the remote TrueNAS system's IP address that contains the NFS share, *{path/to/nfsShare}* is the path to the NFS share on the TrueNAS system, and *{localMountPoint}* is a local directory on the host system configured for the mounted NFS share.
```
Comment (micjohnson777) [inline content/SCALE/Shares/_index.md:901]: suggestion to remove possessive case
```suggestion
In the above example, *{IPaddressOfTrueNASsystem}* is the remote TrueNAS system IP address that contains the NFS share, *{path/to/nfsShare}* is the path to the NFS share on the TrueNAS system, and *{localMountPoint}* is a local directory on the host system configured for the mounted NFS share.
```

**Review by micjohnson777 — COMMENTED** [review]
After the one correction, and if the rest of the article doesn't have possessive case or use the word will, then it is ready to approve!

**Comment by tonyriv3** [comment]
@micjohnson777 
I got rid of all the possessives except for the ones that are verbatim UI tool tips.

**Review by micjohnson777 — APPROVED** [review]
approving this PR!

**Pull request merged**